docs/visualization/visualize_unit_metrics.ipynb

```diff
@@ -107,14 +107,14 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ax.set_xlabel("Depth (Channel #)")
</pre><p>Perhaps add the Standard deviation to this plot so that we get a sense of variability. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>